index.html

```diff
@@ -17,12 +18,23 @@
 <body>
 
  <!-- navigation code here -->
+ <div class= "container top-home">
```

Comment:
Great job on the HTML! I would maybe take out the container class and include it in your container div below